src/styles/main.scss

```diff
@@ -1,3 +1,61 @@
+@import "./variables";
+@import "./normalize";
+
 body {
-  margin: 0;
+  display: flex;
+  width: 706px;
+  margin: 0 auto;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  font-family: "Arial", sans-serif;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $cell-gap;
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    box-sizing: border-box;
```

Comment:
Please move to the * selector and remove it from all other places.

Reply:
Some styles on the browse page disappeared after I moved to * selector